tests/core/test_views_auth.py

```diff
@@ -1,3 +1,72 @@
+from django.contrib.auth.mixins import AccessMixin
+from django.urls import get_resolver, reverse
+from guardian.mixins import LoginRequiredMixin
+
+
 def test_accounts_login(django_app):
     resp = django_app.get("/accounts/login/")
     assert resp.status_code == 200, "Should return a 200 status code"
+
+
+def test_accounts_logout(django_app, superuser):
+    resp = django_app.get(reverse("core:oidc_logout"), user=superuser).follow()
+    assert resp.status_code == 200, "Should return a 200 status code"
+    assert not hasattr(resp.request, "user")
+
+
+def test_all_views_are_secured(django_app):
```

Comment:
this again raises the question how such tests can/should also be run for our plugins